### PR TITLE
Port sparse data test from the old repo

### PR DESCRIPTION
**Pull request by phoebusm**
#### Reference Issues/PRs

Related to https://github.com/man-group/arcticdb-enterprise/issues/120

#### What does this implement or fix?
Porting sparse data related test from the old repo
#### Checklist

<details>
  <summary>
   Checklist for code changes...
  </summary>
 
 - [ ] Have you updated the relevant docstrings, documentation and copyright notice?
 - [ ] Is this contribution tested against [all ArcticDB's features](../docs/mkdocs/docs/technical/contributing.md)?
 - [ ] Do all exceptions introduced raise appropriate [error messages](https://docs.arcticdb.io/error_messages/)?
 - [ ] Are API changes highlighted in the PR description?
 - [ ] Is the PR labelled as enhancement or bug so it appears in autogenerated release notes?
</details>

### CODE REVIEW

File: python/tests/stress/arcticdb/version_store/test_sparse.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+def test_sparse_interleaved(sym, lmdb_version_store):
+    lib = lmdb_version_store
+    df = get_interleaved_dataframe(100)
+    lib.write(sym, df, dynamic_schema=True)
+    dd = lib.read(sym, dynamic_schema=True).data
+
+    assert dd["float"][2] == df["float"][2]
+
+    # TODO: This should be NaN
```
Comment (poodlewars) [inline python/tests/stress/arcticdb/version_store/test_sparse.py:35]: What does this TODO mean? It is NaN?

File: python/tests/stress/arcticdb/version_store/test_sparse.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+def test_sparse_chunked(sym, lmdb_version_store):
+    lib = lmdb_version_store
+    df = get_dataframe_dense_till(100, 30)
+    lib.write(sym, df, allow_sparse=True)
+    dd = lib.read(sym, allow_sparse=True).data
+    assert dd["float"][2] == df["float"][2]
+
+    # TODO: This should be NaN
```
Comment (poodlewars) [inline python/tests/stress/arcticdb/version_store/test_sparse.py:46]: What? No they shouldn't?

Assertion on the values that actually should be NaN

**Review by poodlewars — CHANGES_REQUESTED** [review]

File: python/tests/stress/arcticdb/version_store/test_sparse.py (hunk 2 above)
Comment (phoebusm) [reply python/tests/stress/arcticdb/version_store/test_sparse.py]: Removed

File: python/tests/stress/arcticdb/version_store/test_sparse.py (hunk 1 above)
Comment (phoebusm) [reply python/tests/stress/arcticdb/version_store/test_sparse.py]: Removed

**Review by willdealtry — APPROVED** [review]

**Review by poodlewars — APPROVED** [review]

**Pull request merged**